file_converters/ifcjson/ifc2json5a.py

```diff
@@ -76,9 +76,8 @@ class IFC2JSON5a(common.IFC2JSON):
     }
 
     settings = ifcopenshell.geom.settings()
-    settings.USE_PYTHON_OPENCASCADE = True
-    settings.set(settings.USE_WORLD_COORDS, True)
-    settings.set(settings.EXCLUDE_SOLIDS_AND_SURFACES, False)
+    settings.set("iterator-output", ifcopenshell.ifcopenshell_wrapper.NATIVE)
```

Comment:
```suggestion
    settings.set('dimensionality', ifcopenshell.ifcopenshell_wrapper.CURVES_SURFACES_AND_SOLIDS)
```

The equivalent of EXCLUDE_SOLIDS_AND_SURFACES is called dimensionality now.